### PR TITLE
small wording fixes for Kuzzle commands

### PR DESCRIPTION
**Pull request by benoitvidis**

### CODE REVIEW

File: bin/kuzzle [hunk 1]
```diff
@@ -9,82 +9,84 @@
...
-  .option('--server', 'start an instance without workers')
-  .option('--worker', 'spawn a single set of workers instead of starting a Kuzzle instance')
-  .option('--likeAvirgin', 'reset before start, so that Kuzzle is like a virgin, touched for the very first time')
-  .option('--fixtures <fixtures>', 'allow you to load a fixtures file')
-  .option('--mappings <mappings>', 'allow you to load and apply a mappings file')
-  .description('Start a Kuzzle instance')
+  .description('start a Kuzzle server|worker node')
+  .option('-p, --port <port>', 'Kuzzle port number', parseInt)
+  .option('-s, --server', 'start Kuzzle in server mode (default)')
+  .option('-s, --worker', 'start Kuzzle in worker mode')
```
Comment (stafyniaksacha) [inline bin/kuzzle]: Option `-s` is defined twice. Did you mean `-w` ?

**Comment by stafyniaksacha** [comment]
Despite my remark, LGTM !

File: bin/kuzzle [hunk 2]
```diff
@@ -9,82 +9,83 @@
...
   .action(require('./commands/install'));
 
+// $ kuzzle likeAvirgin
+program
+  .command('likeAvirgin')
+  .description('delete Kuzzle configuration and users from database. Makes Kuzzle look like a virgin, touched for the very first time')
+  .option('--pid <pid>', '[all|integer] the pid to apply the action')
+  .option('--fixtures <fixtures>', 'import some fixtures from file')
+  .option('--mappings <mappings>', 'load and apply mappings from file')
+  .option('--noint', 'non interractive mode, will perform the reset immediately')
```
Comment (scottinet) [inline bin/kuzzle]: nitpicking: interactive

File: bin/kuzzle [hunk 3]
```diff
@@ -9,82 +9,83 @@
...
-  .option('--activate', '*Marks the plugin as "activated" (Kuzzle ignores deactivated plugins)')
-  .option('--deactivate', '*Marks the plugin as "deactivated" (Kuzzle ignores deactivated plugins)')
-  .option('--importConfig <file>', '*Import a configuration from a file for a given plugin')
-  .option('-v, --npmVersion <version>', 'Installs plugin <version> from NPM (work only with --install)')
-  .option('-u, --gitUrl <url>', 'Installs plugin from a GIT repo <url> (work only with --install)')
-  .option('-p, --path <path>', 'Installs a plugin from directory <path> (work only with --install)')
+  .description('manage plugins')
+  .option('    --list', 'list currently installed plugins')
+  .option('    --install', 'if plugin [name] is provided, install it from --npmVersion, --gitUrl or --path, otherwise, (re-)install all listed plugins')
+  .option('    --remove', 'removes plugin [name] from Kuzzle')
```
Comment (scottinet) [inline bin/kuzzle]: nitpicking (old mistake that was never fixed): "remove" instead of "removes"

**Comment by scottinet** [comment]
+1

**Comment by ballinette** [comment]
Why, in the command descriptions, is there sometimes spaces before the options names (`.option('    --list', 'list currently installed plugins'`), and sometime not (`.option('--pid <pid>', '[all|integer] the pid to apply the action')`)?

**Pull request merged**